src/report-export/services/code-pen-report-export-service.tsx

```diff
@@ -44,7 +44,7 @@ class CodePenExportForm extends React.Component<ReportExportFormProps> {
                         editors: '100', // collapse CSS and JS editors
                     })}
                 />
-                <button type="submit" ref={this.buttonRef} />
+                <button type="button" ref={this.buttonRef} />
```

Comment:
Made this change because while running test case its throwing below error
![image](https://github.com/microsoft/accessibility-insights-web/assets/155614445/a727dd38-2bb2-4635-b0d9-634cbb99dced)

and as per the error, stackoverflow had below suggestion
![image](https://github.com/microsoft/accessibility-insights-web/assets/155614445/fcda539b-8620-472d-b309-c7d84a4214dd)

so implemented this, please let me know, because there is not type submit for a button.
Please suggest

Reply:
This error message is expected and fine because it is correct that the submit functionality is not implemented (because the form gets submitted to codepen and gets handled on their end. I'm okay with this error showing up in logs for now.